examples/outputs/alias.rs

```diff
@@ -26,14 +27,16 @@ use iota_client::{
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    // Create a client instance.
     let client = Client::builder()
         .with_node("http://localhost:14265")?
         .with_node_sync_disabled()
         .finish()
         .await?;
 
-    // This example uses dotenv, which is not safe for use in production
-    // Configure your own seed in ".env". Since the output amount cannot be zero, the seed must contain non-zero balance
+    // This example uses dotenv, which is not safe for use in production!
+    // Configure your own seed in the ".env" file. Since the output amount cannot be zero, the seed must contain
```

Comment:
```suggestion
    // Configure your own mnemonic in the ".env" file. Since the output amount cannot be zero, the seed must contain
```
I will try to remove the seed parts from most things, because it will be better to use mnemonics